example/lib/data_sources.dart

```diff
@@ -88,7 +88,8 @@ class DessertDataSource extends DataTableSource {
     desserts = [];
   }
 
-  DessertDataSource(this.context, [sortedByCalories = false]) {
+  DessertDataSource(this.context,
+      [sortedByCalories = false, this.hasRowTaps = false]) {
```

Comment:
This became quite ugly after I added the extra positional optional parameter to it. I think it will be better if named optional parameters are used here. But there will be a lot of files change. What do you think?

Reply:
Why is it ugly? For the example with taps there will be a call to DessertDataSource passing 3 params and the rest will stay the same?

Reply:
Yes, basically filling up a `false` in the middle:

```
      _dessertsDataSource = DessertDataSource(
          context, false, getCurrentRouteOption(context) == rowTaps);
```

And yes to for "everyone else will stay the same". As long as you are okay, then all is well.